README.md

```diff
@@ -295,10 +295,10 @@ This tutorial shows several visualization approaches for 3D image during transfo
 This folder shows how to run the comprehensive Auto3DSeg pipeline with minimal inputs and customize the Auto3Dseg modules to meet different user requirements.
 
 #### <ins>**Self-Supervised Learning**</ins>
-##### [self_supervised_pretraining](./self_supervised_pretraining/ssl_train.ipynb)
+##### [self_supervised_pretraining](self_supervised_pretraining/vit_unetr_ssl/ssl_train.ipynb)
 This tutorial shows how to construct a training workflow of self-supervised learning where unlabeled data is utilized. The tutorial shows how to train a model on TCIA dataset of unlabeled Covid-19 cases.
 
-##### [self_supervised_pretraining_based_finetuning](./self_supervised_pretraining/ssl_finetune.ipynb)
+##### [self_supervised_pretraining_based_finetuning](self_supervised_pretraining/vit_unetr_ssl/ssl_finetune.ipynb)
 This tutorial shows how to utilize pre-trained weights from the self-supervised learning framework where unlabeled data is utilized. This tutorial shows how to train a model of multi-class 3D segmentation using pretrained weights.
 
```

Comment:
@finalelement please also add the `swinunetr_finetune.ipynb` to the README.md in the root in a separate PR. Thanks!